code/modules/holodeck/computer.dm

```diff
@@ -92,8 +92,13 @@ GLOBAL_LIST_INIT(typecache_holodeck_linked_floorcheck_ok, typecacheof(list(/turf
 
 /obj/machinery/computer/holodeck/LateInitialize()//from here linked is populated and the program list is generated. its also set to load the offline program
 	linked = GLOB.areas_by_type[mapped_start_area]
-	bottom_left = locate(linked?.x, linked?.y, src.z) // SKYRAT EDIT: Fixing CI with our maps
-
+	// SKYRAT EDIT BEGIN: Fixing CI with our maps
+	bottom_left = locate(linked?.x, linked?.y, src.z)
+	if(!bottom_left)
+		log_world("No matching holodeck area found - Skyrat Edition (addition?)")
+		qdel(src)
+		return
+		// SKYRAT EDIT END
```

Comment:
I'll go ask on /tg/ why it doesn't fail for them, but it fails for us. I'll ping you on Discord about it too. Did you happen to have the logs on hand, to make my life easier?